packages/app-rfi/src/core/utils/appState.js

```diff
@@ -0,0 +1,257 @@
+
+import * as Yup from "yup";
+import { useFormik } from "formik";
+import React, { useEffect, useState } from "react";
+
+import { variants } from "../../components/appVariants";
+import { CertInfo } from "../../components/steps/CertInfo.js";
+import { Success } from "../../components/steps/Success.js";
```

Comment:
`           7:25  error    Unexpected use of file extension "js" for "../../components/steps/Success.js"  import/extensions`
Error thrown during build linting. Lines 7 and 8 both have the issue.